<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailedStudySession</class>
 <widget class="QWidget" name="DetailedStudySession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DetailedStudySession</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>363</width>
     <height>262</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="11" column="0" colspan="2">
     <widget class="QPushButton" name="joinButton">
      <property name="text">
       <string>Join</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>98</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLabel" name="limitEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="sizeLabel">
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLabel" name="sizeEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>98</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLabel" name="descriptionEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLabel" name="locationEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>98</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="2">
     <widget class="QPushButton" name="leaveButton">
      <property name="text">
       <string>Leave</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="locationLabel">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="limitLabel">
      <property name="text">
       <string>Limit:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLabel" name="endTimeEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="endTimeLabel">
      <property name="text">
       <string>End Time:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLabel" name="startTimeEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="hostEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="hostLabel">
      <property name="text">
       <string>Host:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="startTimeLabel">
      <property name="text">
       <string>Start Time:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="subjectLabel">
      <property name="text">
       <string>Subject:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLabel" name="subjectEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLabel" name="dateEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
